src/PrestaShopBundle/Form/Admin/Type/CustomMoneyType.php

```diff
@@ -44,15 +74,60 @@ public static function getExtendedTypes(): iterable
      */
     public function configureOptions(OptionsResolver $resolver)
     {
+        $defaultCurrencyIso = Currency::getIsoCodeById($this->defaultCurrencyId);
```

Comment:
It is weird to inject an id and fetch with a legacy method IMO.

Maybe put this logic into a separate service that will provide the legacyIsoCode if not set into form options.